### PR TITLE
Initial Setup

### PR DESCRIPTION
**Pull request by atharvabhanage02**
This PR includes : 

- Navbar Component
- Sidebar Component
- Login and Signup Pages
- Authentication ( User can login and logout to the system using Test credentials ) 

Files to look for : 

- Components : `src/Components` (Navbar and Sidebar)
- Context : `src/Context/Auth/auth-context.js`
- Pages : `src/Pages` ( Explore , Login , Signup )

Here is the Deploy Preview : https://deploy-preview-1--cricplay.netlify.app/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,12 +10,15 @@
...
+    "react-icons": "^4.4.0",
+    "react-router": "^6.3.0",
+    "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
     "uuid": "^8.3.2"
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "CI= react-scripts build",
```
Comment (MarkVed17) [inline package.json:21]: Curious to know about this.

File: public/index.html [hunk 2]
```diff
@@ -10,6 +10,20 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--  -->
```
Comment (MarkVed17) [inline public/index.html]: Could add something to this comment or maybe remove it entirely.

File: src/Components/Sidebar/Sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+export const Sidebar = () => {
+  const { pathname } = useLocation();
+  return (
+    <div>
+      <aside className="sidebar">
+        <div className="pages">
+          <Link
+            className={`category-names ${
+              pathname === "/" ? "active-tab" : "white"
+            } `}
```
Comment (MarkVed17) [inline src/Components/Sidebar/Sidebar.jsx:19]: There's a better way to style the `Active` link.
Check this out https://reactrouter.com/docs/en/v6/components/nav-link

File: src/Pages/Signup/Signup.jsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                placeholder="Enter Valid Email ID"
+              />
+            </div>
+            <div class="password">
+              <p for="">Password</p>
+              <input type="password" class="pswrd-validate" />
+            </div>
+            <div class="reset-details">
+              <div>
+                <input type="checkbox" class="checkbox" />I accept all the Terns
```
Comment (MarkVed17) [inline src/Pages/Signup/Signup.jsx]: terms*

**Review by MarkVed17 — APPROVED** [review]
I understand this is the `initial setup` but you can add a `title` & `favicon` as well.
Rest all looks good to proceed.

File: public/index.html (hunk 2 above)
Comment (atharvabhanage02) [reply public/index.html]: Removed it .

**Pull request merged**